dist/lifx.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'lifx-http-api' {
+  interface ILifxOptions {
+    bearerToken: string;
+  }
+
+  const LifxHTTPAPI: (options: ILifxOptions) => void;
```

Comment:
The goal of this file is to allow for 
```typescript
const lifx = require('lifx-http-api');
new lifx({
  …
});
```
The return type of a constructor must be `void` in TypeScript

![image](https://cloud.githubusercontent.com/assets/1683595/24071537/beb2ef32-0b91-11e7-8bee-39c34032f00b.png)